docs/examples/en/controls/OrbitControls.html

```diff
@@ -87,7 +87,7 @@ <h3>[property:Boolean autoRotate]</h3>
 		<h3>[property:Float autoRotateSpeed]</h3>
 		<p>
 			How fast to rotate around the target if [page:.autoRotate] is true. Default is 2.0, which equates to 30 seconds
-			per orbit at 60fps.<br> Note that if [page:.autoRotate] is enabled, you must call [page:.update]
+			per orbit at 60fps. To unlock [page:.autoRotate] from framerate, utilise 'controls.autoRotateSpeed = 15 * clock.getDelta();', you must import the clock module and set up its' constructor first, Example : 'let clock = new Clock( {autoStart : true} )'.  <br> Note that if [page:.autoRotate] is enabled, you must call [page:.update]
```

Comment:
Sorry, but I do not think this is appropriate for the docs. The docs document the API only.